src/screens/ComponentLibraryScreen.tsx

```diff
@@ -291,6 +297,126 @@ function ComponentLibraryScreen() {
       <Backdrop bgClass="bg-slate-700/50">
         <TripHeader />
       </Backdrop>
+      <Title>Form Header</Title>
+      <View className="flex flex-row items-center justify-between px-12">
+        <IconButton
+          size={IconSize.Small}
+          icon="ChevronLeftIcon"
+          colour={useColours()['ghost']}
+          bgClass="bg-tuatura"
+          onPress={() =>
+            formHeaderStep !== 0 ? setFormHeaderStep(formHeaderStep - 1) : null
+          }
+        />
+        <IconButton
+          size={IconSize.Small}
+          icon="ChevronRightIcon"
+          colour={useColours()['ghost']}
+          bgClass="bg-tuatura"
+          onPress={() =>
+            formHeaderStep <= 4 ? setFormHeaderStep(formHeaderStep + 1) : null
+          }
+        />
+      </View>
+      <Backdrop bgClass="bg-tuatura/75" fullWidth>
+        <FormHeader
+          leftComponent={
+            formHeaderStep === FormStep.TripDetails ? (
```

Comment:
Instead of adding state and more moving parts to this screen shall we just show the different variants stack on top of eachother?

Reply:
Yeah